src/browser-installer/install.ts

```diff
@@ -6,7 +6,7 @@ import _ from "lodash";
 export const installBrowser = async (
     browserName?: string,
     browserVersion?: string,
-    { force = false, installWebDriver = false } = {},
+    { force = false, shouldInstallWebDriver = false, shouldInstallUbuntuPackages = true } = {},
```

Comment:
Lets pretend you want to install browser.
Does it mean you also need webdriver? No, because you might want to run "devtools" protocol
Does it mean you also need ubuntu packages? Yes (check "if it is ubuntu" is inside)

I can change both to "true" though